src/lamplib/tasks/download.py

```diff
@@ -0,0 +1,132 @@
+import logging
+import os
+import shutil
+import subprocess
+import sys
+import urllib.request
+
+from context import Context
+
+class Downloader():
```

Comment:
should we be using [abc](https://docs.python.org/3/library/abc.html) here? (I actually don't know what the best-practices are these days). Either way we're python3.7+ so we don't need the parens just `class Downloader` 💅